cmd/agent/app/integrations_nix.go

```diff
@@ -3,7 +3,7 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-2019 Datadog, Inc.
 
-// +build !windows
+// +build !windows !darwin
```

Comment:
should read `!windows,!darwin` (comma-separated) since it's `AND` we want to express here (a space means `OR`)

Reply:
ah thanks, explains the ci failures :)